timelock-server-benchmark-cluster/src/main/java/com/palantir/atlasdb/timelock/benchmarks/server/config/TimelockBenchmarkServerConfig.java

```diff
@@ -41,4 +42,11 @@ public TimeLockRuntimeConfiguration runtime() {
         return runtime;
     }
 
+    public int threadPoolSize() {
+        return (int) (Math.max(256, 32 * Runtime.getRuntime().availableProcessors()) * 0.5);
```

Comment:
what on earth? 😂 

Can we just do:

    Math.max(256, 32 / 2 * Runtime.getRuntime().availableProcessors())